nft_ingester/src/parsers/gummyroll.rs

```diff
@@ -154,6 +163,44 @@ pub async fn gummyroll_change_log_event_to_database(
             .await
             .map_err(|db_err| IngesterError::StorageWriteError(db_err.to_string()))?;
     }
+
+    // If and only if the entire path of nodes was inserted into the `cl_items` table, then insert
```

Comment:
Thanks, I was trying to think about partial failures, duplicated records, and out of order records, since I'm not actually sure what kind of performance issues are possible yet in our overall system.